packages/clerk-js/src/ui/hooks/useNavigateToFlowStart.ts

```diff
@@ -3,7 +3,8 @@ import { useRouter } from '../router';
 export const useNavigateToFlowStart = () => {
   const router = useRouter();
   const navigateToFlowStart = async () => {
-    const to = '/' + router.basePath + router.flowStartPath;
+    const to = router.indexPath;
```

Comment:
Could you briefly, our discoveries and why this is a "safe" change ?

Reply:
In the current modal setup, the first element is associated with the '/' path. Previously, clicking the back button in the modal's sidebar navigation would navigate back to the path constructed by '/' + router.basePath + router.flowStartPath, since the modal always opened with the first element selected. 

Now, with the ability to open the modal on a specific element, the router.flowStartPath is not necessarily the path for the first element anymore. That's why we should now navigate to router.indexPath instead of the previously used router.flowStartPath.